### ISSUE TITLE
Allow selecting files in torrent, useful for TV shows

### ISSUE THREAD

**Issue by Kabouik** [trianglecirclexyz/cthulhu#3]
Currently `cthulhu` automatically uses `peerflix -a`, which downloads all files. It is a problem for TV shows whose episodes are all bundled into the same torrent. `peerflix -l` seems to cover exactly that case (although it doesn't allow multi-selection, unfortunately), and same with `webtorrent -s` (a little bit less convenient). Wouldn't it make sense to make an option (and perhaps use `-a` as default) so that users can use `-l`/`-s` instead and decide which episode to stream when several are bundled together?

Ideally there could be a prompt asking whether to download all files or just one after a torrent has been selected, because the user can't know in advance which torrents would be available. Specifying the option when executing `cthulhu` would skip that prompt.

**Comment by trianglecirclexyz** [comment]
You're absolutely right that `-l` works better for shows compared to `-a`. I just pushed a change that adds the `-l`/`--list` flags that replace `-a` with `-l` in `peerflix`. `-a` is still used by default (although I am planning on adding a config file to allow users to change all the defaults). I haven't added an automatic prompt yet, but maybe I will later on. 

Also, I made it so when users use `--list`/`-l`, there is a loop that asks them if they want to continue watching after the player closes. If they pick yes, it brings up the list again, making it easier to binge watch tv shows or whatever.
